### ISSUE TITLE
Android client double touch act as mouse one click

### ISSUE THREAD

**Issue by khaleids** [FreeRDP/FreeRDP#4571]
Hello,

 I'm using Android FreeRDP client, 
in Microsoft RDP on Android, one touch act as, mouse one click, 
and double touch act as mouse double click, 
problem are in FreeRDP one touch act as nothing,  double touch act as mouse one click, 
so is there options to use it same as Microsoft RDP??
I'm using normal touch, not pointer

**Comment by khaleids** [comment]
Version Information
aFreeRDP Version

2.0.0-rc1-387-g1d2903c (2.0.0-dev2)

System Version

5.0

**Comment by akallabeth** [comment]
@khaleids You need to run `./scripts/android-build-freerdp.sh` to compile the native dependencies and then edit the `aFreeRDP` project in android studio. (be aware, `NDK 16` currently does not work, try an older one)
The base project is quite old, so feel free to fix(or ignore) any warning you encounter ^^

**Comment by khaleids** [comment]
there is no same as mouse double click in Android client ?? nobody uses Android client ??!!!

**Comment by akallabeth** [comment]
@khaleids The mode you want is currently not implemented, see the help option for usage.

**Comment by khaleids** [comment]
@akallabeth I fixed this issue, I want to push but, I cant is said
fatal: unable to access 'https://github.com/FreeRDP/FreeRDP.git/': The requested URL returned error: 403

**Comment by khaleids** [comment]
@awakecoding 
I fixed this issue, I want to push but, I cant, it's said
fatal: unable to access 'https://github.com/FreeRDP/FreeRDP.git/': The requested URL returned error: 403

**Comment by khaleids** [comment]
Command = git push

**Comment by khaleids** [comment]
I tested on emulator, it works perfect, but on physical device, when click on server to connect it crash, I think it's related to building issue, so I want to push it to github repo, because you hAve cloud test system.

**Comment by hardening** [comment]
@khaleids you can't push directly on FreeRDP, clone the repo, push on your copy and then submit a Pull Request.

**Comment by khaleids** [comment]
@hardening Yes thank you, i forked ;)

**Comment by khaleids** [comment]
I'm trying to run build scripts but it didnt work on my os it's Windows 10 Enterprise 64
C:\aFreeRDP\FreeRDP>scripts\android-build-freerdp.sh --ndk C:\Users\xxx\Downloads\android-ndk-r15c-windows-x86_64\android-ndk-r15c\ --sdk C:\Users\xxx\AppData\Local\Android\Sdk\

**Comment by khaleids** [comment]
it's show this 
![image](https://user-images.githubusercontent.com/23697904/38987509-5b472b70-43d0-11e8-9fe8-60c3d7cca24f.png)

**Comment by khaleids** [comment]
@akallabeth So which OS do you use, to developing FreeRDP?

**Comment by khaleids** [comment]
Yes I installed Ubuntu 16,04 LTS 64-bit but it gives an error when I run the script 
![image](https://user-images.githubusercontent.com/23697904/39056849-0cb2fbcc-44b8-11e8-8aa4-16ed0373b17f.png)
